completions/pyenv.fish

```diff
@@ -18,5 +18,6 @@ end
 
 complete -f -c pyenv -n '__fish_pyenv_needs_command' -a '(pyenv commands)'
 for cmd in (pyenv commands)
-  complete -f -c pyenv -n "__fish_pyenv_using_command $cmd" -a "(pyenv completions $cmd)"
+  complete -f -c pyenv -n "__fish_pyenv_using_command $cmd" -a \
+    "(pyenv completions (commandline -opc)[2..-1])"
```

Comment:
Wouldn't it be better to check length of `(commandline -opc)`like zsh completion's doing? In case of `(commandline -opc)` == 2, I guess it should return `(pyenv commands)` instead of `(pyenv completions (command -opc)[2..-1])`. Please see also zsh's implementation.

https://github.com/yyuu/pyenv/blob/f36d735e27a7619da02a4bb45c9b4a4958b5a888/completions/pyenv.zsh#L11-L15